src/host/usbh.c

```diff
@@ -246,8 +246,8 @@ static uint8_t _usbh_ctrl_buf[CFG_TUH_ENUMERATION_BUFSIZE];
 
 // Control transfer: since most controller does not support multiple control transfer
 // on multiple devices concurrently. And control transfer is not used much except enumeration
-// We will only execute control transfer one at a time.
-struct
+// We will only execute control transfer one at a time.CFG_TUSB_MEM_SECTION TU_ATTR_ALIGNED(4096)
+CFG_TUSB_MEM_SECTION struct
```

Comment:
please revert this change, only the buffer needed to be in the usb section, the xfer info is not

Reply:
As you have requested I have reverted this commit but, I still have an issue with this as my port will not function without it,  I will look into this further and let you know what I find...  I suspect it is some sort of caching problem.

New commit will follow shortly...

Cheers,
Pete